core/commands/configure_command.rb

```diff
@@ -3,8 +3,14 @@
 require_relative 'base_command'
 require_relative '../services/aws_service'
 
+require 'open3'
+
```

Comment:
What is the point of these empty lines and the empty line before the require statement?